subsys/net/lib/zperf/zperf_shell.c

```diff
@@ -191,7 +191,7 @@ static int cmd_setip(const struct shell *sh, size_t argc, char *argv[])
 		if (zperf_get_ipv6_addr(argv[start + 1], argv[start + 2],
 					&shell_ipv6) < 0) {
 			shell_fprintf(sh, SHELL_WARNING,
-				      "Unable to set IP\n");
+				      "Unable to set %s address (%s disabled)\n", "IPv6", "IPv4");
```

Comment:
Why pass const string as argument instead having it as part of the fmt itself?

Reply:
To save memory, the format strings are RO and can be re-used if they are same saving memory.